source/shared/core_conn.cpp

```diff
@@ -527,7 +527,7 @@ void core_sqlsrv_get_client_info( sqlsrv_conn* conn, _Out_ zval *client_info TSR
 // Properly escaped means that any '}' should be escaped by a prior '}'.  It is assumed that 
 // the value will be surrounded by { and } by the caller after it has been validated
 
-bool core_is_conn_opt_value_escaped( const char* value, size_t value_len )
+bool core_is_conn_opt_value_escaped( _In_ const char* value, _Inout_ size_t value_len )
 {
```

Comment:
For SAL, does it not matter that value is changed (line 575)? Given it's just a pointer increment